### PR TITLE
Release/0.7.0

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,6 +43,8 @@ bin_linux     = $(linux_dir)/$(bin_name)
 bin_darwin    = $(darwin_dir)/$(bin_name)
 bin_windows   = $(windows_dir)/$(bin_name)
 
+gox           = "github.com/mitchellh/gox"
```
Comment (jbeemster) [inline Makefile:46]: Why the change to how `gox` was being pulled in?

File: config/cluster.json.sample [hunk 2]
```diff
@@ -61,7 +61,7 @@
       {
         "name": "Elasticity Bootstrap Action",
         "scriptBootstrapAction": {
-          "path": "s3://snowplow-hosted-assets-us-east-1/common/emr/snowplow-ami4-bootstrap-0.2.0.sh",
+          "path": "s3://snowplow-hosted-assets-us-east-1/common/emr/snowplow-ami5-bootstrap-0.1.0.sh",
```
Comment (jbeemster) [inline config/cluster.json.sample:64]: Is this still needed?

Comment (istreeter) [reply config/cluster.json.sample:64]: 🤷

The purpose of `cluster.json.sample` is to demo the format of the configuration file.  So whether it's needed or not, there should be an example of a bootstrap action.  This one seemed reasonable, because we own it and it's publicly visible.

Do you have any suggestions for an alternative bootstrap action?  I guess I could make it fictious, like `s3://my-bucket/path/to/bootstrap.sh`

File: Makefile (hunk 1 above)
Comment (istreeter) [reply Makefile:46]: Because I got this error when running `make`:

```
GO111MODULE=on go get -u github.com/mitchellh/gox
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

Comment (jbeemster) [reply Makefile:46]: I think you can borrow some inspiration from my other Makefiles (https://github.com/snowplow-devops/metrics-relay/blob/master/Makefile#L38-L40).  Have hit this issue and these ones work well up to Go 1.17

Comment (istreeter) [reply Makefile:46]: The make file your linked to does not work for me.  If I run `make all` in metrics-relay I get this:

```
GO111MODULE=on go install github.com/mitchellh/gox@latest
mkdir -p build/compiled
GO111MODULE=on CGO_ENABLED=0 gox -osarch=linux/amd64 -output=build/output/linux/metrics-relay .
/bin/sh: line 1: gox: command not found
make: *** [Makefile:51: cli-linux] Error 127
```

I'm running go 1.17.6.

The make file I wrote here works, and I think it's future proof beyond 1.17.
